packages/x-components/src/x-modules/search-box/store/module.ts

```diff
@@ -9,17 +10,22 @@ import { SearchBoxXStoreModule } from './types';
  */
 export const searchBoxXStoreModule: SearchBoxXStoreModule = {
   state: () => ({
-    query: ''
+    query: '',
+    status: 'initial'
```

Comment:
Since we have status property in other modules, maybe we should name it state or something else. Or change the status in the other modules to be a state machine.

Reply:
You have the same problem with `state`. If I have to choose between the 2 I would leave status. In the other modules it refers to a request status, in this one as we don't have a request it refers to the input status.

Reply:
if it refers to input status why don't just called it that? `inputStatus`